libs/@local/eslint-config/generate-ignore-patterns.cjs

```diff
@@ -29,25 +114,43 @@ module.exports = (workspaceDirPath) => {
     .relative(monorepoRoot, workspaceDirPath)
     .replace(/\\/g, "/")}/`;
 
-  const sharedPatternsFromPrettierignore = match
+  const sharedPatternsFromPrettierIgnore = match
```

Comment:
I'm not sure about this rename as it's called `.prettierignore` not `.prettier-ignore` or something like that. I find the new name more readable, though.